test/performance/observed_concurency_test.go

```diff
@@ -199,7 +199,7 @@ func TestObservedConcurrency(t *testing.T) {
 		}
 	}
 
-	if err = testgrid.CreateTestgridXML(tc); err != nil {
```

Comment:
any chance for this perf test to also consume loadgenerator?

Reply:
it needs to perform some additional steps to each response in the request, which the load generator currently does not support. So, unfortunately, we cannot use that

Reply:
how about extend loadgenerator to allow customized handler? We should design for customization.